src/test/java/org/elasticsearch/rest/action/admin/indices/upgrade/UpgradeTest.java

```diff
@@ -246,6 +243,7 @@ public static void runUpgrade(HttpRequestBuilder httpClient, String index, Strin
         assertEquals(200, rsp.getStatusCode());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
nothing....just that I went over this class because it used JsonPath#evaluate which I was changing and then I went wild on fixing compiler warnings.... can revert :)